templates/repo/create.tmpl

```diff
@@ -8,7 +8,11 @@
 					{{.i18n.Tr "new_repo"}}
 				</h3>
 				<div class="ui attached segment">
-					{{template "base/alert" .}}
```

Comment:
This will break not to show other error messages

Reply:
Are there other error messages?

Reply:
If subbmiting form has any validation errors